fapolicy_analyzer/ui/system_trust_database_admin.py

```diff
@@ -123,4 +124,5 @@ def on_trust_selection_changed(self, trust):
 
     def on_addBtn_clicked(self, *args):
         if self.selectedFile:
-            self.file_added_to_ancillary_trust(self.selectedFile.path)
+            for sfile in self.selectedFile:
+                self.file_added_to_ancillary_trust(sfile.path)
```

Comment:
These files should be added in a single changeset, so you'll have to update the `file_added_to_ancillary_trust` event and associated handler to handle multiple file paths as an argument.  Similar to my comment from the add/delete trust in the ATDA file.